pkg/agent/util/iptables/iptables.go

```diff
@@ -145,18 +150,25 @@ func (c *Client) EnsureRule(table string, chain string, ruleSpec []string) error
 		if err := ipt.Append(table, chain, ruleSpec...); err != nil {
 			return fmt.Errorf("error appending rule %v to table %s chain %s: %v", ruleSpec, table, chain, err)
 		}
+		klog.V(2).Infof("Added rule %v to table %s chain %s of %s protocol", ruleSpec, table, chain, p.String())
```

Comment:
can you switch to structured logging (`InfoS` / `ErrorS`) for all the log messages that you add or modify?

Reply:
Thank you for catching this. I did pay attention to `klog` but this `klog.V(2)` somehow blinds me...